swdc

```diff
@@ -14,6 +14,12 @@ source "${DIR}/functions.sh"
 # Need to be in docker folder for docker-compose
 cd "${DIR}"
 
+if [[ "${1}" == "" ]];
+then
+  swdc generate-command-list
```

Comment:
```suggestion
  source modules/base/generate-command-list.sh
```
Would be easier

Reply:
Sure it would be easier but I think it's cleaner like this

Reply:
swdc will fail, when swdc is not in path